cms/templates/base.html

```diff
@@ -5,13 +5,14 @@
 from django.utils.translation import get_language_bidi
 from django.template.defaultfilters import escapejs
 import json
-
-dir_rtl = 'rtl' if get_language_bidi() else 'ltr'
 %>
+<%def name="dir_rtl()">
+  <% return 'rtl' if get_language_bidi() else 'ltr' %>
+</%def>
```

Comment:
In my perusal of Mako files we seem to use the block Python forms. However I don't know if this means we have an actual preferance for it. However, the `<%def` line just looks wrong to me, because I'm not used to it.